src/storage/storage_backend.rs

```diff
@@ -160,6 +164,32 @@ pub trait StorageBackend<U: Sync + Send + Debug>: Send + Sync + Debug {
     /// [`Metadata`]: ./trait.Metadata.html
     async fn metadata<P: AsRef<Path> + Send + Debug>(&self, user: &Option<U>, path: P) -> Result<Self::Metadata>;
 
+    /// Returns the `Md5` for the given file.
+    ///
+    /// Whether or not you want to implement the md5 method yourself,
+    /// or you want to let your StorageBackend make use of the below
+    /// default implementation, you must still explicitly enable the
+    /// feature via the `supported_features` method.
```

Comment:
`supported_features` can actually be a doc link: [supported_features](crate::storage::StorageBackend::supported_features)

Reply:
Don't take my word on the syntax.

Reply:
I did and it worked 😆